sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -105,15 +105,22 @@ static ipv6_addr_t *_add_addr_to_entry(gnrc_ipv6_netif_t *entry, const ipv6_addr
     }
     else {
         if (!ipv6_addr_is_link_local(addr)) {
-#ifdef MODULE_GNRC_NDP_ROUTER
-            /* New prefixes MAY allow the router to retransmit up to
-             * GNRC_NDP_MAX_INIT_RTR_ADV_NUMOF unsolicited RA
-             * (see https://tools.ietf.org/html/rfc4861#section-6.2.4) */
+#if defined(MODULE_GNRC_NDP_ROUTER) || defined(MODULE_GNRC_SIXLOWPAN_ND_ROUTER)
             if ((entry->flags & GNRC_IPV6_NETIF_FLAGS_ROUTER) &&
                 (entry->flags & GNRC_IPV6_NETIF_FLAGS_RTR_ADV)) {
-                entry->rtr_adv_count = GNRC_NDP_MAX_INIT_RTR_ADV_NUMOF;
                 mutex_unlock(&entry->mutex);    /* function below relocks mutex */
+#ifdef MODULE_GNRC_NDP_ROUTER
+                /* New prefixes MAY allow the router to retransmit up to
+                 * GNRC_NDP_MAX_INIT_RTR_ADV_NUMOF unsolicited RA
+                 * (see https://tools.ietf.org/html/rfc4861#section-6.2.4) */
+                entry->rtr_adv_count = GNRC_NDP_MAX_INIT_RTR_ADV_NUMOF;
                 gnrc_ndp_router_retrans_rtr_adv(entry);
```

Comment:
Still double RA on the 6LoWPAN interface of a 6LBR, since there is one send out here and below.